config/translations/en/module.download.yml

```diff
@@ -3,6 +3,9 @@ arguments:
     module: 'Module or modules to be enabled should be separated by a space'
 options:
     latest: 'Default to download most recent version'
+    module-path: 'The path of the contrib project'
```

Comment:
@darol100 in fact I thought when I started to use the same rather than duplicate... because you/we ask for the same thing.